dnf-behave-tests/features/command-aliases.feature

```diff
@@ -0,0 +1,104 @@
+Feature: Tests for command aliases availability
+
+
+Scenario: "nonexistent" is not an alias for any dnf command
+   When I execute dnf with args "nonexistent -h"
+   Then the exit code is 1
+    And stderr contains "No such command: nonexistent"
+
+
+Scenario Outline: "<alias>" is an alias for "<command>"
+   When I execute dnf with args "<alias> -h"
```

Comment:
The scenario doesn't even test the `<command>` part of the examples... Looking at the output it seems every help text starts with `usage: dnf <command> ` (followed by the list of options). Perhaps we could check for that?

Reply:
Hey, that's a good idea! I've been thinking that checking the functionality is too much work, so I would only check that it's an existing alias, but I haven't thought of checking the help for the main command. I've added it, thanks.